### ISSUE TITLE
Load ICPSR crosswalk names into cde_agencies

### ISSUE THREAD

**Issue by harrisj** [18F/crime-data-api#584]
After we raised the issue that the default public names for agencies are often confusing (ie, in New Jersey there is Camden County Police Department and Camden County for the sheriff), we got the go-ahead to use the ICPSR crosswalk if we want to. This process would involve a few steps:

- [x] Load the latest ICPSR data into the database (from the TSV)
- [x] Copy over the name fields to cde_agencies. Since these are all-caps, this would likely mean doing a title-case transformation with some exemptions for abbreviations (like PD)
- [x] We might just add in lat/lng and zip codes for agencies if we are loading this data

**Comment by harrisj** [comment]
One wrinkle is that we probably can't put the crosswalk TSV in the repo because of #468

**Comment by harrisj** [comment]
After a discussion with the FBI this morning, we have supplied them with a CSV of agency names that are just the following rule:

1. If a city, append " Police Department" to the `pub_agency_name`
2. If a county, append " County Sheriff's Department" to the `pub_agency_name` if doesn't already have that County in its name
3. Otherwise, just use the `pub_agency_name`

This covers 95% of what we need and the rest could be hand-edited as needed by them (for instance, it's just the Miami-Dade County Police Department and not a sheriff or such. The idea behind this PR is to load in two supplementary files that are used to add several new fields to cde_agencies:

1. `agency_name` - the edited name that can be copy-edited by the FBI
2. `short_name` - the original pub_agency_name is useful as a short name for the agency

In addition, I have also added some code to import a few useful fields from the ICPSR 2012 crosswalk:

1. `icpsr_zip` - the zip code of the agency address
2. `icpsr_lat` - the latitude of the agency
3. `icpsr_lng` - the longitude

Note that locations are merely approximate. Spot-checking a few town agencies, it seems to place them outside city limits generally and is likely in the centroid of the agency jurisdiction. So, it won't show you where the HQ for that agency is. But it will help to place the agency on a state map.

Second note: not all agencies have lat/lng. Have a fallback option as appropriate.
